docs/content.zh/docs/dev/table/sql/show.md

```diff
@@ -118,7 +118,7 @@ tEnv.executeSql("SHOW CURRENT DATABASE").print();
 // create a table
 tEnv.executeSql("CREATE TABLE my_table (...) WITH (...)");
 // show tables
-tEnv.executeSql("SHOW TABLES").print();
+tEnv.executeSql("SHOW TABLES FROM default_database LIKE 'm%'").print();
```

Comment:
Could you revert changes to the general example? I think we don't need to show complex examples here.

Reply:
done